internal/start/start.go

```diff
@@ -750,7 +750,7 @@ EOF
 					fmt.Sprintf("DB_PORT=%d", dbConfig.Port),
 					"DB_USER=supabase_admin",
 					"DB_PASSWORD=" + dbConfig.Password,
-					"DB_NAME=" + dbConfig.Database,
```

Comment:
`realtime` schema should be initialised inside the postgres database. We probably need to check with @filipecabaco if there's additional config for separating the control plane database.

```
psql (16.3, server 15.6)
Type "help" for help.

_supabase=> \dn
        List of schemas
    Name    |       Owner
------------+-------------------
 _analytics | postgres
 _realtime  | postgres
 _supavisor | postgres
 public     | pg_database_owner
 realtime   | postgres
(5 rows)

_supabase=> \dt realtime.*
                       List of relations
  Schema  |       Name        | Type  |          Owner
----------+-------------------+-------+-------------------------
 realtime | messages          | table | supabase_realtime_admin
 realtime | schema_migrations | table | supabase_admin
 realtime | subscription      | table | supabase_admin
(3 rows)
```